h/traversal/roots.py

```diff
@@ -284,7 +284,11 @@ def __init__(self, request):
         self.user_svc = self.request.find_service(name="user")
 
     def __getitem__(self, userid):
-        user = self.user_svc.fetch(userid)
+        try:
+            user = self.user_svc.fetch(userid)
```

Comment:
I wonder if all the other places that call `fetch` handle this case properly? We could perhaps avoid the issue by making `fetch` return `None` in that case, so that callers only have to handle two possible outcomes rather than three.